rpc/src/module/stats.rs

```diff
@@ -49,15 +49,14 @@ impl<CS: ChainStore + 'static> StatsRpc for StatsRpcImpl<CS> {
     }
 
     fn get_peers_state(&self) -> Result<Vec<PeerState>> {
+        // deprecated
```

Comment:
The deprecated info can be added to the RPC doc (in rpc/json/rpc.json and then `make gen-doc`).